packages/amplify_datastore/test/amplify_datastore_stream_controller_test.dart

```diff
@@ -275,7 +274,7 @@ void main() {
     expect((element.model as Post).created, equals(parsedDate));
     expect(element.deleted, equals(false));
     expect(element.version, equals(1));
-    expect(element.lastChangedAt, equals(1));
+    expect(element.lastChangedAt, equals(1624492860));
```

Comment:
nit: this is the original code, think that we should get the expected value from `json` instead hardcoding it here.

Reply:
Sure, just didn't feel like changing too much in this PR.